ckine/figures/figure6.py

```diff
@@ -70,14 +70,14 @@ def makeFigure():
     affComp(ax[0])
     calc_plot_specificity(ax[7], 'NK', df_spec, df_act, ckines, ckineConc_)
     calc_plot_specificity(ax[8], 'T-helper', df_spec, df_act, ckines, ckineConc_)
-    global_legend(ax[8], Spec=True, Mut=True)
```

Comment:
How about get rid of the predicted EC50’s in B? We don’t talk about the model until later anyway. A parallel coordinates plot might be especially effective here, too: https://pandas.pydata.org/pandas-docs/version/0.9.0/visualization.html